starwars/src/components/StartWarCharacterList.js

```diff
@@ -1,15 +1,10 @@
 import React from 'react'
-import Table from './Table/Table'
 import StarWarCharacter from './StarWarCharacter'
 
-export default function starWarCharacterList({ data }) {
-  // return (
-  //   <Table data={data} />
-  // )
-
+export default function starWarCharacterList({ starWarCharacters }) {
   return (
     <ul>
-       {data.map((char, i) => (
+       {starWarCharacters.map((char, i) => (
          <StarWarCharacter character={char}  key={`${char.name} ${i}`}/>
        ))}
     </ul>
```

Comment:
- Great job on the sprint challenge Chinedu. 
- You successfully completed the MVP by fetching data from 3rd party API and displayed it on the webpage. You have properly split functionality into smaller components and passed necessary data as props.
- Your project is styled properly and design looks good.
- You have also gone above , the user its able to expand and see detail information about the character . 
- That spinner make a better user experience . Great job on implemented it!
Your code is clean and Your git workflow is fluid.
Overall:
* Able to fetch and display data (MVP)
* Good answers
* Spinner - when data is loading 
* Clean and modular code with multiple components
* Implemented styling
* Responsive design
* Fluid git workflow